org.omg.sysml.interactive/src/org/omg/sysml/interactive/SysMLInteractive.java

```diff
@@ -191,29 +194,34 @@ public SysMLInteractiveResult eval(String input, boolean isAddResource) {
 		}
 	}
 	
+	private Resource getDummyResource() {
+		if (this.dummyResource == null) {
+			this.dummyResource = this.createResource("dummy" + SYSML_EXTENSION);
+			this.dummyResource.getContents().add(SysMLFactory.eINSTANCE.createNamespace());
+		}
```

Comment:
Why do we need to add a SysML namespace to the dummy resource?  It will not affect anything but I do not see your intention to add it.

Reply:
It is probably not strictly necessary, but it makes the resource a legal KerML/SysML resource with an empty root namespace. It also prevents the system from trying to autoload content for the resource.

Reply:
Thank you.  That helps me understand it.